src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,13 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: %s\n"
+            + "Please exit the application by clicking the exit button (i.e., the button marked with an 'X' icon).";
+    public static final String FILE_OPS_PERMISSION_ERROR_MESSAGE = "Could not save data to file: "
+            + "Insufficient Permissions for %s.\n\n"
+            + "Please exit the application by clicking the exit button (i.e., the button marked with an 'X' icon).\n"
```

Comment:
It's been a while so I might have forgotten the reason; why the user has to exit the application?